src/gtfs/read.rs

```diff
@@ -204,7 +208,7 @@ impl<'de> ::serde::Deserialize<'de> for RouteType {
     }
 }
 
-#[derive(Serialize, Deserialize, Debug, Clone)]
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, Hash, PartialEq)]
```

Comment:
why do you need Eq and Hash?

Reply:
for HashSet I suppose. It does not compile without